frontend/simple/js/state.js

```diff
@@ -15,7 +15,7 @@ import debounce from 'lodash/debounce'
 
 Vue.use(Vuex)
 var store // this is set and made the default export at the bottom of the file.
-          // we have it declared here to make it accessible in mutations
+// we have it declared here to make it accessible in mutations
```

Comment:
This is unclear now. I recommend reverting this how it was, but instead adding an extra linebreak so that it's clear that this comment refers to `var store`, and the following comment refers to `const state`